app/src/androidTest/java/au/com/shiftyjelly/pocketcasts/analytics/experiments/ExperimentProviderTest.kt

```diff
@@ -0,0 +1,42 @@
+package au.com.shiftyjelly.pocketcasts.analytics.experiments
+
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+class ExperimentProviderTest {
```

Comment:
Can you add tests cases for `null` and unknown variation names?

Reply:
Good idea, I have made this change.